rootfs-builder/README.md

```diff
@@ -106,16 +65,18 @@ must be met:
    ```
 
    Note:
-   This requirement is specifically when using Clear Containers runtime
+   This requirement is specific to the Clear Containers runtime
```

Comment:
Suggest adding a period after "runtime" but that will cause "see" on the next line to be capitalized.